src/main/java/world/bentobox/bentobox/api/flags/Flag.java

```diff
@@ -603,16 +603,14 @@ public Flag build() {
             // If no clickHandler has been set, then apply default ones
             if (clickHandler == null) {
                 switch (type){
-                case PROTECTION:
-                    clickHandler = new CycleClick(id);
-                    break;
-                case SETTING:
+                    case SETTING:
                     clickHandler = new IslandToggleClick(id);
                     break;
                 case WORLD_SETTING:
                     clickHandler = new WorldToggleClick(id);
                     break;
-                default:
+                    case PROTECTION:
+                    default:
                     clickHandler = new CycleClick(id);
                     break;
                 }
```

Comment:
Beware of the indentation here. Switch cases are indented this way
```java
switch (type) {
    case SETTING:
        // code
        break;
    case PROTECTION:
    default:
        // code
        break;
}
```
etc.

Reply:
Well, it is weird to provide case statement, if it falls through default.

Reply:
The indentation came from the IDE which was weird - yes. Also, the point of this is to show that PROTECTION is the default option. It could be removed, but then you may think that PROTECTION was forgotten. So that's the idea. It's a stylistic thing that was recommended to me.

Reply:
I fixed the indentation and added a comment on the default.